### PR TITLE
Site build

### PR DESCRIPTION
**Pull request by cattre**

### CODE REVIEW

File: beers.php [hunk 1]
```diff
@@ -22,17 +25,48 @@
...
+                                        <br>
+                                        <?php echo "Style: {$beer['style']}"; ?>
+                                        <br>
+                                        <?php echo "ABV: {$beer['abv']}"; ?>
+                                    </details>
+                                </article>
+                            <?php } endforeach; ?>
+                        </div>
+                    </section>
+                <?php endforeach; } ?>
```
Comment (hb020202) [inline beers.php:69]: There is probably a reason for this but on this line the curly bracket is after the end statement but on line 66 it's before

File: beers.php [hunk 2]
```diff
@@ -22,17 +25,48 @@
 <html lang='en'>
 	<head>
 		<title>A world of beers</title>
-		<link rel='stylesheet' type= 'text/css' href='normalize.css'>
+        <link href='https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Mansalva&display=swap' rel='stylesheet'>
+        <link rel='stylesheet' type= 'text/css' href='normalize.css'>
```
Comment (hb020202) [inline beers.php:29]: Indent these two lines

**Review by hb020202 — APPROVED** [review]
Two super small comments!

File: beers.php (hunk 1 above)
Comment (cattre) [reply beers.php:69]: Yes, just based on what it's closing (one of the foreach loops is nested, the other isn't).

File: beers.php (hunk 2 above)
Comment (cattre) [reply beers.php:29]: Weird, doesn't look like that on mine - they'll all indented! Oh well :)

**Pull request merged**